src/app/components/Loader/styles.module.scss

```diff
@@ -1,19 +1,23 @@
 @import 'variables/colors';
 
+@import 'variables/zIndex';
```

Comment:
its better to have a `variables/_variables.scss` and store there all the variables like this in there